server/controllers/getDroplets/index.ts

```diff
@@ -18,18 +20,23 @@ const getDroplets =
 
     let row = null;
     try {
-      row = db
-        .query<
-          { points: number; change: number; place: number },
-          [string]
-        >('SELECT points + points_l1 + points_l2 as points, change, place FROM user_points_public WHERE address = ? LIMIT 1')
-        .get(address);
+      const result = await db.query<{
+        points: number;
+        change: number;
+        place: number;
+      }>(
+        'SELECT (points + points_l1 + points_l2)::int as points, change::int as change, place::int as place FROM user_points_public WHERE address = $1 LIMIT 1',
```

Comment:
it may fail due to overflow `(points + points_l1 + points_l2)::int` here

Reply:
Good catch, changed to `::bigint`

Reply:
why can't we use numeric/decimal?

Reply:
Now using numeric and changed response type to string